automation_tools/satellite6/upgrade/__init__.py

```diff
@@ -95,9 +95,9 @@ def satellite6_upgrade(admin_password=None):
     # Rebooting the system again for possible errors
     if os.environ.get('RHEV_SATELLITE') or os.environ.get('SAT_HOST'):
         reboot(120)
-    # Stop the service again which started in restart
-    run('katello-service stop')
     if to_version == '6.1':
+        # Stop the service again which started in restart
```

Comment:
can you please add a comment to say that these are already done by satellite-installer in 6.2?